pm/sender.go

```diff
@@ -151,12 +151,14 @@ func (s *sender) validateTicketParams(ticketParams *TicketParams, numTickets int
 		return err
 	}
 
+	ev := ticketEV(ticketParams.FaceValue, ticketParams.WinProb)
 	// validate sender
-	if err := s.validateSender(info); err != nil {
-		return err
+	if ev.Cmp(big.NewRat(0, 1)) > 0 {
```

Comment:
Just a minor thing, I don't think executing this entire function is necessary if `ev == 0` because 

- totalEV will also be 0 and will never exceed `maxEV`
- faceValue is 0 and will never exceed `maxFaceValue`

So if I'm not mistake we could move the `TicketParams.ExpirationBlock` checks before this block

i.e.

```
... TicketParams expiration checks ... 

if ev.Cmp(big.newRat(0,1)) <=0 { return nil }

...valitedateSender, maxEv and maxFaceVal checks...
```

Reply:
Fixed in e067db8